plugins/single-instance/src/platform_impl/linux.rs

```diff
@@ -89,7 +87,7 @@ pub fn init<R: Runtime>(f: Box<SingleInstanceCallback<R>>) -> TauriPlugin<R> {
 
 pub fn destroy<R: Runtime, M: Manager<R>>(manager: &M) {
     if let Some(connection) = manager.try_state::<ConnectionHandle>() {
-        let dbus_name = format!("org.{}.SingleInstance", dbus_id(manager.config()));
+        let dbus_name = format!("org.{}.SingleInstance", dbus_id(&manager.config()));
```

Comment:
This `&` shouldn't be necessary, right? (Since config() already returns `&Config`)
Same for line 38.